### ISSUE TITLE
Feature request - required and ng-pattern validation for ui-select

### ISSUE THREAD

**Issue by baskaranr** [johannesjo/ng-fab-form#92]

**Comment by johannesjo** [comment]
I agree, that this would be a nice feature. I don't know when I will have time for that though. In the meantime feel free to submit a pr. The best way to do it, would be probably to allow the `ngFabFormsValidationsDirective` to take parameters for restrict and the name. This would allow to apply the validations directive to any element, e.g. something like that:
```JavaScript
ngFabFormsValidationsDirective('ui-select',{restrict:'ea'});
```
https://github.com/johannesjo/ng-fab-form/blob/master/src/ng-fab-form-validations-directive-f.js

**Comment by baskaranr** [comment]
I tried validation for ui-select by adding 

`angular.module('ngFabForm')
    .directive('uiSelect', ['ngFabFormValidationsDirective', function (ngFabFormValidationsDirective)
    {
        'use strict';

        return ngFabFormValidationsDirective;
    }]);
`.
`<div class="form-group">
	<label class="col-sm-4 control-label" for="gender">Gender</label>
	<div class="col-sm-8">
		<ui-select ng-model="item.Gender" name="gender" required>
			<ui-select-match>
				{{$select.selected.Text | limitTo : 20 : 0}}
				<span ng-if="$select.selected.Text.length>20">...</span>
			</ui-select-match>
			<ui-select-choices repeat="lookupitem.Id as lookupitem in lookup.Gender | filter: { Text: $select.search }">
				<div ng-bind-html="lookupitem.Text | highlight: $select.search"></div>
			</ui-select-choices>
		</ui-select>
	</div>
</div>`.

But the validation is not working.
